_posts/2020-03-19-letsencrypt-dns01.md

```diff
@@ -0,0 +1,214 @@
+---
+title: "DNS01 Challenge Provider for Let’s Encrypt Issuer using Google CloudDNS"
+layout: post
+date: 2020-03-19 12:05
+headerImage: false
+tag:
+- kubernetes
+- google cloud
+- ingress
+- letsencrypt
+- cert-manager
+category: blog
+author: kosyanyanwu
+description: Configuring DNS01 Challenge Provider using Google CloudDNS for Let's Encrypt issuer setup with cert-manager
+---
+## Introduction
+[DNS01 challenge](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge) asks you to prove that you control the DNS for your domain name by putting a specific value in a TXT record under that domain name. It is harder to configure than HTTP-01, but can work in scenarios that HTTP-01 can’t. It also allows you to issue wildcard certificates. After Let’s Encrypt gives your ACME client a token, your client will create a TXT record derived from that token and your account key, and put that record at `_acme-challenge.<YOUR_DOMAIN>`. Then Let’s Encrypt will query the DNS system for that record. If it finds a match, you can proceed to issue a certificate!
```

Comment:
- As a reader, I may want to know what is hard to configure. A short explanation of the challenge will keep the reader more engaged.

Reply:
I think I will remove it instead. I simply got it from the docs directly